issues.md

```diff
@@ -0,0 +1,38 @@
+Reporting issues
+================
+
+By default, all Issues logged in Github are meant to be considered to be a _bug_, meaning, _a software problem affecting the stability of the product_. This includes functionalities that run without error, but do something unexpected which also works against the system's stability.
+
+Priority labels
+---------------
+In September 2016, Ministry Brands helped define priority levels for issues based on an overall Service Level Agreement (SLA). We use the labels P1, P2, P3 and P4 define the overall _priority_ of resolution on the issue.
```

Comment:
I don't think we should include the history of how we came to this. That's better left for the commit message.